blog/docs/articles/demystifying-activator-on-path.md

```diff
@@ -0,0 +1,325 @@
+
+**Note:** if the KPA metric Requests Per Second(RPS) is used then the utilization factor is 0.75.
+
+The `observedPanicValue` is the calculated average value seen during the panic window for the concurrency metric.
+The panic threshold is configurable (default 2) and expresses the ratio of desired versus available pods.
+
+After we enter panic mode in order to exit we need to have enough capacity for a period that is equal to the stable window size.
+That also means that autoscaler will try to get enough pods ready in order to increase the capacity.
+Also note here that when operating outside the panic mode the autoscaler does not use `dpcc` but a similar quantity:
+`dspc := math.Ceil(observedStableValue / spec.TargetValue)` which is based on metrics during the stable period.
+
+To quantify the idea of enough capacity and to deal with bursts of traffic we introduce the notion of the Excess Burst Capacity(EBC) that needs to be >=0.
+It is defined as:
+
+```
+EBC = TotalCapacity - ObservedPanicValue - TargetBurstCapacity(TBC).
+```
+
+`TotalCapacity` is calculated as ready_pod_count*revision_total. The default TBC is set to 200. 
+Now with the above defaults and given the fact that a request needs to stay around for some time in order concurrency metrics to show enough load, it means you can't get EBC>=0 with a hello-world example.
+The latter which is often confusing for the newcomer as the Knative service never enters serve mode. 
+In the next example we will show the lifecycle of a Knative service that also moves to serve mode and how ebc is calculated in practice. 
+
+Here is a service that has a target concurrency of 10 and tbc=10:
+
+```
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: autoscale-go
+spec:
+  template:
+    metadata:
+      annotations:
+        autoscaling.knative.dev/target: "10"
+        autoscaling.knative.dev/target-burst-capacity: "10"
+    spec:
+      containers:
+      - image: ghcr.io/knative/autoscale-go:latest
+```
+
+The scenario we are going to demonstrate is deploye the ksvc, let it scale down to zero and then send traffic for 10 minutes.
+We then collect the logs from the autoscaler and visualize the EBC, ready pods and panic mode over time.
+The graphs are shown next.
+
+![Excess burst capacity over time](/blog/articles/images/ebc.png)
+![Ready pods over time](/blog/articles/images/readypods.png)
+![Panic mode over time](/blog/articles/images/panic.png)
+
+
+Let's describe in detail what we see above. Initially when the ksvc is deployed there is no traffic and one pod is created by default for verification reasons.
+
+Until the pod is up we have:
+```bash
+$ kubectl get sks
+NAME                 MODE    ACTIVATORS   SERVICENAME          PRIVATESERVICENAME           READY     REASON
+autoscale-go-00001   Proxy   2            autoscale-go-00001   autoscale-go-00001-private   Unknown   NoHealthyBackends
+```
+When the pod is up we have:
+
+```bash
+$ oc get po
```

Comment:
nit: probably should stick with either kubectl or oc for these commands

Reply:
Oups :) Yeah makes sense.